opennms-doc/guide-admin/src/asciidoc/index.adoc

```diff
@@ -172,6 +172,12 @@ include::text/operation/newts/configuration.adoc[]
 include::text/operation/newts/cassandra-jmx.adoc[]
 include::text/operation/newts/cassandra-newts-jmx.adoc[]
 
+[[ga-opennms-operation-daemon-config-files]]
+=== Daemon Configuration Files
+include::text/operation/daemon-config-files/introduction.adoc[]
```

Comment:
Hi @RangerRick 

They are here https://github.com/OpenNMS/opennms/pull/890 and here https://github.com/OpenNMS/opennms/pull/891

I don't know how to handle this, sorry.